src/utils/listing.js

```diff
@@ -60,6 +65,8 @@ export function originToDAppListing(originListing) {
     name: originListing.title,
     pictures: originListing.media ? originListing.media.map(medium => medium.url) : [],
     price: originListing.price.amount,
+    boostValue: originListing.commission && originListing.commission.amount,
```

Comment:
If originListing.commission is defined, the data coming from originListing.commission.amount is guaranteed to be a string. Can we convert to float here so that the getBoostLevel method can always assume it gets a float ?
Maybe something like:
`boostValue: originListing.commisson ? parseFloat(originListing.commisson.amount) : 0.0`

And same for line below.

Reply:
Good call - updated